Sources/ComposableArchitecture/Documentation.docc/Tutorials/BuildingSyncUps/06-SyncUpDetail/EditingAndDeletingSyncUp-03-code-0005.swift

```diff
@@ -25,10 +25,6 @@ struct SyncUpDetail {
     case destination(PresentationAction<Destination.Action>)
     case doneEditingButtonTapped
     case editButtonTapped
-    @CasePathable
-    enum Alert {
```

Comment:
this was also brought up in an email. we have some extra `Alert` actions hanging around that should have been deleted after introducing the `Destination` reducer.